src/ocl/utilocl.cpp

```diff
@@ -80,8 +80,8 @@ float Im2d2ColGPU(const Handle& handle,
 
     int data_size_bound = c * in_h * in_w;
 
-    int data_size_bound_pack = type == miopenInt8x4 ? data_size_bound * 4 : data_size_bound;
-    int im_offset_pack       = type == miopenInt8x4 ? im_offset / 4 : im_offset;
+    int data_size_bound_pack = data_size_bound;
```

Comment:
Can we use original variables `data_size_bound` and `im_offset` everywhere now?

Reply:
@amberhassaan Kept to minimize the diff and probability of typos. Can resolve this in my next PR if you insist, just let me know.

Reply:
Not insisting but would be nice to address this in a future PR. A future reader might be confused as to what's going on.